Solutions/Palo Alto - XDR (Cortex)/Detection Queries/WildFire Malware Detection.yaml

```diff
@@ -0,0 +1,33 @@
+id: 961672e7-15db-4df1-9bab-dc4f032b9b6f
+name: Palo Alto WildFire Malware Detection Rule
```

Comment:
Remove the word Rule from the name. All the detections are Analytical Rules by default.